packages/manager/modules/billing/src/components/cancel-commitment/cancel-commitment.controller.js

```diff
@@ -1,12 +1,23 @@
 export default class ServicesActionsCtrl {
   /* @ngInject */
-  constructor($http, $translate) {
+  constructor($http, $translate, atInternet) {
     this.$http = $http;
     this.$translate = $translate;
+    this.atInternet = atInternet;
   }
 
   cancelCommitment() {
     this.isLoading = true;
+
+    // track action
+    if (this.confirmCancelTracking) {
```

Comment:
`this.confirmCancelTracking` should never be null no? (Regarding https://github.com/ovh/manager/pull/4434/files#diff-c4d38bcecd6d308fa11ecfa985ca213f72b7193742725b207ec67c45ba04896aR13)

Reply:
It should not indeed. I added this test if in an other implementation (VPS for example), we don't put it.